src/wazuh_db/wdb_global.c

```diff
@@ -924,12 +927,12 @@ int wdb_global_delete_agent_belong(wdb_t *wdb, int id) {
         return OS_INVALID;
     }
 
-    if (wdb_stmt_cache(wdb, WDB_STMT_GLOBAL_DELETE_AGENT_BELONG) < 0) {
+    if (wdb_stmt_cache(wdb, WDB_STMT_GLOBAL_DELETE_ALL_AGENT_BELONG) < 0) {
```

Comment:
Now that we are adding the **ALL** word to make it clearer, the method name `wdb_global_delete_agent_belong()` should also be updated

Reply:
I think the names of the methods are clear. I have adapted the queries names to:
```
WDB_STMT_GLOBAL_DELETE_AGENT_BELONG
WDB_STMT_GLOBAL_DELETE_TUPLE_BELONG
```
to better represent that the first one completely removes all the entries of an agent and the second one only removes one entry.